core-container/entry.sh

```diff
@@ -242,51 +230,68 @@ function run_kcp() {
 
 function run_init() {
     echo "--< Starting init >--"
-    wait_kcp_ready
     set_provider_adapters
-    kubestellar init --local-kcp false --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW
-    kubectl ws root
+    kubestellar init --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW 
     touch ready
     echo "***READY***"
     sleep infinity
 }
 
-
 function run_mailbox_controller() {
     echo "--< Starting mailbox-controller >--"
     wait-kubestellar-ready
+    # TODO: remove the kcp dependency
```

Comment:
Maybe be a little more explicit about _when_ to do? Perhaps something like `# TODO in S3: ...`?

Reply:
# TODO: remove kcp dependency in PR3 once we switch to kube-bind.

Reply:
I do not see the update:

<img width="952" alt="Screenshot 2023-11-17 at 12 09 31 AM" src="https://github.com/kubestellar/kubestellar/assets/14296719/ab46e2be-1906-4f61-a4a3-904a0b93216d">

Even when I go to your fork on GitHub and view the PR branch there, I do not see the update: https://github.com/effi-ofer/edge-mc/blob/e21244a42bec4d53560fb9170e9b49dcc849d85c/core-container/entry.sh#L243
(commit e21244a42bec4d53560fb9170e9b49dcc849d85c is what your branch "PR2" currently points to).